commerce_coordinator/apps/commercetools/sub_messages/tasks.py

```diff
@@ -398,6 +397,13 @@ def _prepare_segment_event_properties(in_order, total_amount, return_line_item_r
             logger.info(f'[CT-{tag}] payment {psp_payment_id} not refunded, '
                         f'sending Slack notification, message id: {message_id}')
 
+    else:
```

Comment:
[Question] are the enrollment code orders the only ones without payment id set? can there be any other case? 

suggestion: to be consistent with frontend checks for enrollment code, can we do this?
```suggestion
    else if psp_payment_id is None and order.total_price == 0:
```

frontend check [reference](https://github.com/frontastic-developers/customer-twou/blob/master/packages/checkout/frontend/components/ui/organisms/thank-you/index.tsx#L26)

Reply:
I agree, but i dont think adding `if psp_payment_id is None and order.total_price == 0:` creates any complex code, its still straightforward but consistent with what we have on frontend.